modules/api/main.tf

```diff
@@ -9,78 +9,79 @@
 // TODO - Refactor using
 //      https://github.com/terraform-google-modules/terraform-google-vm
 resource "random_string" "random" {
-  length = 8
-  lower = true
-  upper = false
+  length  = 8
+  lower   = true
+  upper   = false
   special = false
   keepers = {
-    otpapi_template_tags = join("", sort(local.otpapi_template_tags)),
-    otpapi_template_machine_type = local.otpapi_template_machine_type,
-    otpapi_template_source_image = local.otpapi_template_source_image,
+    otpapi_template_tags          = join("", sort(local.otpapi_template_tags)),
+    otpapi_template_machine_type  = local.otpapi_template_machine_type,
+    otpapi_template_source_image  = local.otpapi_template_source_image,
     vm_platform_api_image_version = var.vm_platform_api_image_version,
-    vm_startup_script = md5(file("${path.module}/scripts/instance_startup.sh"))
-    vm_flag_preemptible = var.vm_flag_preemptible
+    vm_platform_api_image_version = var.vm_platform_api_image_version,
+    vm_startup_script             = md5(file("${path.module}/scripts/instance_startup.sh"))
+    vm_flag_preemptible           = var.vm_flag_preemptible
```

Comment:
We need to include the API version metadata here
```
api_v_major
api_v_minor
...
```
As changes to those values should trigger an API redeployment.